.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -2,21 +2,33 @@
 name: Feature
 about: Suggest an idea for this project
 title: 'MODULE: Neues Feature'
-labels: ''
+labels: feature
 assignees: ''
 
 ---
 
-Neben der Adresse soll neu auch ...
+Als PERSON_DIE_DAMIT_ZU_TUN_HAT,
+möchte ich EIN_FEATURE_HABEN,
+um EIN_ZIEL_ZU ERREICHEN
+
+Neu soll ...
 
 # Mockup
 
 hier folgt ein Mockup ...
 
 # Tech-Spec
 
-- https://github.com/hitobito/hitobito
+- sinngebende Zielrichtung
+- besondere/bekannten Bedingungen und Grenzfälle
+- technische Einschränkungen
+- Ein- und Ausgabeformate
 
 # ToDo
 
-- [ ] Schritte für Umsetzung
+- [ ] Migration erstellen
+- [ ] Domainklasse erstellen oder anpassen
+- [ ] Model/View/Controller anpassen
+- [ ] Specs schreiben
+- [ ] Kunde wegen Übersetzungen informieren
+- [ ] CHANGELOG-Eintrag hinzufügen
```

Comment:
Gemäss https://github.com/hitobito/hitobito/wiki/Definition-of-Done müsste mindestens hier noch "Mit angemessener Rolle durchgeklickt" stehen